docker/docker-entrypoint.sh

```diff
@@ -62,7 +62,8 @@ if [ "$1" = 'timesketch' ]; then
   tsctl add_user -u "$TIMESKETCH_USER" -p "$TIMESKETCH_PASSWORD"
 
   # Run the Timesketch server (without SSL)
-  exec `bash -c "/usr/local/bin/celery -A timesketch.lib.tasks worker --uid nobody --loglevel info & /usr/local/bin/tsctl runserver -h 0.0.0.0 -p 5000"`
+  #exec `bash -c "/usr/local/bin/celery -A timesketch.lib.tasks worker --uid nobody --loglevel info & /usr/local/bin/tsctl runserver -h 0.0.0.0 -p 5000"`
+  exec `bash -c "gunicorn -b 127.0.0.1:80 --log-file - --timeout 120 timesketch.wsgi:application & /usr/local/bin/tsctl runserver -h 0.0.0.0 -p 5000"`
```

Comment:
You are running 2 webservers here, this should be something like:
```
exec `bash -c "/usr/local/bin/celery -A timesketch.lib.tasks worker --uid nobody --loglevel info & gunicorn -b 127.0.0.1:80 --log-file - --timeout 120 timesketch.wsgi:application"`
```

```tsctl runsever``` is only for development

Reply:
Sorry, I should have mentioned this was not intended to be merged yet.  Playing with it a bit, and I tend to commit more often than I should.  Sorry. :(

Reply:
Heh, no worries :) Let me know when you are ready.